content/en/hosting/4.x/production/google-cloud/_index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Production Hosting CHT 4.x - Google Cloud Platform"
+linkTitle: "Google Cloud Platform"
+weight: 10
+aliases:
+   - /hosting/4.x/production/docker/google-cloud/
+description: >
+   Production Hosting CHT 4.x - Google Cloud Platform
+---
+This for a step by step to migrate or deploy CHT 4.x instance and is the recommended solution  on Google Cloud Platform.
```

Comment:
```suggestion
How to deploy the CHT 4.x on Google Cloud Platform.
```

Reply:
This will include steps to migrate to GCP as it'll have narrow references to properly send data to what GCP calls "persistent disks", and "storage provisioners". Should that portion belong somewhere else?

Essentially, our goal was to write this so Muso could perform their exact tasks by reading through one set of docs and then later on abstract it based on community questions.